python/phase-2.md

```diff
@@ -0,0 +1,830 @@
+# Python SDK - Phase 2
+
+The Python SDK will be implemented in three loosely defined phases:
+
+* Phase 1 - Initial project, activity worker, and client
+* Phase 2 - Workflow worker, local activities, async activity completion, etc
+* Phase 3 - Deterministic workflow sandbox, replayer, test suite (some parts may be combined with phase 2)
+
+This proposal is for phase 2. It is intentionally loosely defined and all aspects are subject to discussion/change.
+
+## Overview
+
+Following up from phase 1 and known implementation changes since then, this phase will implement the actual workflow
+worker/code and other advanced features.
+
+A simple example of a workflow definition and worker run:
+
+```python
+from datetime import timedelta
+from temporalio import activity, workflow
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+@activity.defn
+async def say_hello(name: str) -> str:
+    return f"Hello, {name}!"
+
+@workflow.defn
+class MyWorkflow:
+    @workflow.run
+    async def run(arg1: str) -> str:
+        return await workflow.execute_local_activity(
+            say_hello, arg1, start_to_close_timeout=timedelta(seconds=5)
+        )
+
+
+async def main(stop_event: asyncio.Event):
+    client = await Client.connect("http://localhost:7233")
+    async with Worker(
+        client,
```

Comment:
Is `client` the right name here. Technically it is, but semantically you are actually developing a "host" (aka an app that serves workflow worker functionality).
Can this be named "connection" or similar?